docs/runbooks/disaster_recovery/switching_to_dr.md

```diff
@@ -0,0 +1,144 @@
+   1. Set up org infra to push containers to the alternative region.
+   2. Tweak the build scripts to push containers to the alternative region ECR in management account.
+   3. This will be improved once decisions are made about replicating Container images to a secondary region.
+
+6. **WebOps support is recommended during DR**. Though not neccesary, it is strongly advised to reach out to a webops engineer when running this runbook.
+
+## Runbook
+
+The runbook below outlines the steps to be taken. It is important to follow this in the correct order. It is recommended that this is done from a local machine in order to provision these resources, for speed.
+
+### Approximate time to restore service
+
+During initial testing the process took approvimately 45 minutes to 1 hour. This is likely increase on a production Database, due to its size.
+
+### High level approach
+
+requires 4 major tasks:
+
+- [Failover to eu-west-2](#failover-to-eu-west-2)
+  - [Context](#context)
+  - [Prerequisites](#prerequisites)
+  - [Caveats and improvements](#caveats-and-improvements)
+  - [Runbook](#runbook)
+    - [Approximate time to restore service](#approximate-time-to-restore-service)
+    - [High level approach](#high-level-approach)
+    - [Notes on replacement values](#notes-on-replacement-values)
+    - [1. Provision region level resources](#1-provision-region-level-resources)
+    - [2. Pagerduty Setup for DB alerts](#2-pagerduty-setup-for-db-alerts)
+  - [3. Provision environment level resources](#3-provision-environment-level-resources)
+  - [4. Set up CI ingress and commit changes](#4-set-up-ci-ingress-and-commit-changes)
+
+### Notes on replacement values
+
+Use the following as replacement values, depending on the scenario:
+
+- `<account_name>` refers to relevant aws account name e.g. `development, preproduction, or production`
+- `<prod_or_non_prod>` refers to type of alert i.e `Production` for production alerts, otherwise `Non-Production` for Development or Preproduction alerts.
+- `<my-workspace>` refers to the ephemeral environment workspace to select in development account; `preproduction` or `production` in those accounts respectively.
+
+### 1. Provision region level resources
+
+1. In the opg-lpa repository folder, run `direnv allow` if it doesn’t autorun `tfswitch`.
+2. Go to `terraform/region` folder.
+3. Select workspace at region level e.g.:`export TF_workspace=<account_name>`.
+4. Set `breakglass` role; this is only for non development accounts: `export TF_VAR_default_role=breakglass`.
+5. Perform an initialisation of the workspace  `aws-exec vault identity — terraform init`.
+6. In the `terraform.tfvars.json’ under the “accounts" -> “development", change “dr_enabled” value to true and save.
+7. Run a plan on region: `aws-vault exec identity — terraform plan`.
+8. Check contents of the plan.
```

Comment:
anything to look out for in particular or that would be undesireable?

Reply:
Just general issues like failed resource deployments. we saw little issues once we resolved any region specific problems.

Reply:
```suggestion
8. Check contents of the plan. things to look out for include: 
     - no destruction of resources in eu-west-1
     - approx 83 created resources
```